test/Microsoft.AspNetCore.Mvc.FunctionalTests/RazorPagesTest.cs

```diff
@@ -249,6 +249,21 @@ public RazorPagesTest(MvcTestFixture<RazorPagesWebSite.Startup> fixture)
             Assert.Equal("/Login?ReturnUrl=%2FHelloWorldWithAuth", response.Headers.Location.PathAndQuery);
         }
 
+        [Fact]
+        public async Task AuthorizePage_AllowAnonymousForSpecificPages()
+        {
+            // Arrange
+            var url = "/Admin/Login";
+            
+            // Act
+            var response = await Client.GetAsync(url);
+
+            // Assert
+            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+            var content = await response.Content.ReadAsStringAsync();
+            Assert.Equal("Login Page", content);
+        }
+
```

Comment:
You know what I was add it, after that I delete it 😄 , coz I didn't proper guideline for `var` tests.

Sure I will add it once the CI builds complete